### PR TITLE
[ADD] added new module the orders history in POS

### PR DESCRIPTION
**Pull request by GabbasovDinar**

### CODE REVIEW

File: pos_orders_history/models/pos_orders_history_models.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+from odoo import fields, models, api
+
+
+CHANNEL = "pos_orders_history"
+
+
+class PosConfig(models.Model):
+    _inherit = 'pos.config'
+
+    orders_history = fields.Boolean("Orders History", help="Show all orders list in POS", default=False)
```
Comment (yelizariev) [inline pos_orders_history/models/pos_orders_history_models.py]: Make it true by default

File: pos_print_method/doc/index.rst [hunk 2]
```diff
@@ -1,6 +1,6 @@
-==================
- POS Print Method
-==================
+====================
+ POS Orders History
```
Comment (yelizariev) [inline pos_print_method/doc/index.rst]: mistaken changes I guess

**Comment by yelizariev** [comment]
@ilmir-k @GabbasovDinar 
Shall we merge it?

**Comment by GabbasovDinar** [comment]
@yelizariev it's doesn't finish. Ask the @ArtyomLosev

**Pull request merged**